app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -893,6 +895,39 @@ class HomeActivityTest {
     }
   }
 
+  @Test
+  fun testHomeActivity_promotedStoryHasScalableWidth() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_UPTIME_MILLIS)
+    storyProgressTestHelper.markInProgressSavedFractionsStory0Exp0(
+      profileId = profileId1,
+      timestampOlderThanOneWeek = false
+    )
+    logIntoUserTwice()
+    launch<HomeActivity>(createHomeActivityIntent(internalProfileId1)).use {
+      it.onActivity { activity ->
+        testCoroutineDispatchers.runCurrent()
+        scrollToPosition(position = 1)
+        onView(
+          atPositionOnView(
+            recyclerViewId = R.id.home_recycler_view,
+            position = 0,
+            targetViewId = R.id.promoted_story_list_recycler_view
+          )
+        ).check { _, _ ->
+          val promotedStoryCard =
+            activity.findViewById<LessonThumbnailImageView>(R.id.lesson_thumbnail)
+          val pixels = promotedStoryCard?.width?.toFloat()
```

Comment:
For this & the next variable, the names are a bit generic. Suggest renaming them to be more specific as to what they are actually representing ("pixels" is just a unit, and "size" could represent any number of things).

Reply:
Updated variable name.